app/controllers/message_drafts/signature_requests_controller.rb

```diff
@@ -46,11 +46,13 @@ def set_message_draft
     end
 
     def signers_scope
-      Current.tenant.signer_group.users.order(name: :asc)
+      signer_groups_ids = (Current.tenant.signer_group.users.map(&:user_group) + [Current.tenant.signer_group]).map(&:id)
```

Comment:
Tu sa mi zda, ze by to malo ist asi lahsie. Nechceme proste vsetky skupiny z tenanta typu SignerGroup?

Reply:
Mozno dava zmysel Current.tennat.signer_groups vytvorit

Reply:
Nie, SignerGroup je len jedna a zvysne su UserGroups pouzivatelov, ktori maju rolu podpisovatela